spec/requests/citizens/check_answers_spec.rb

```diff
@@ -60,7 +60,7 @@
 
     it 'displays the name of the firm' do
       subject
-      expect(response.body).to include(HTMLEntities.new.encode(firm.name))
```

Comment:
Removing the encoding?

Reply:
yep - it doesn't work, becaue most entities get written out properly, e.g. `&amp;` or `&lt;`, for some reason apostrophes don't get written as `&apos;` but as `&#39;`. I've got another little PR that deals with that.